packages/web3-react/package.json

```diff
@@ -0,0 +1,52 @@
+{
+  "name": "@reef-knot/web3-react",
+  "version": "0.0.1",
+  "main": "dist/cjs/index.js",
+  "module": "dist/esm/index.js",
+  "types": "dist/esm/index.d.ts",
+  "license": "MIT",
+  "homepage": "https://github.com/lidofinance/reef-knot",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/lidofinance/reef-knot.git",
+    "directory": "packages/web3-react"
+  },
+  "bugs": {
+    "url": "https://github.com/lidofinance/reef-knot/issues"
+  },
+  "publishConfig": {
+    "registry": "https://registry.npmjs.org/",
+    "access": "public"
+  },
+  "scripts": {
+    "build": "rollup -c",
+    "dev": "rollup -c -w"
+  },
+  "devDependencies": {
+    "@ethersproject/providers": "^5.4.5",
+    "@types/ua-parser-js": "^0.7.36"
+  },
+  "dependencies": {
+    "@gnosis.pm/safe-apps-web3-react": "0.6.8",
+    "@ledgerhq/iframe-provider": "0.4.2",
+    "@lido-sdk/constants": "^1.6.0",
```

Comment:
Here we have lido-sdk as dependencies instead of peerDependencies, is this also expected?

Reply:
For now, yes